### ISSUE TITLE
Model in eval mode during training

### ISSUE THREAD

**Issue by giuliadascenzi** [ShengcaiLiao/TransMatcher#8]
Hello @ShengcaiLiao,
thank you very much for your work.
I do have a question: which are the reasons why you are keeping your model (the feature extractor part) in eval mode during training? In the train epoch you do: 
`self.model.eval()
 self.criterion.train()`

Thank you again.
